docs/source/howtos/add_version.rst

```diff
@@ -0,0 +1,36 @@
+How To Add a Version To an Instrument
+=====================================
+
+ResINS strives to model all :term:`versions<version>` of any given :term:`INS`
+:term:`instrument` - old, new, and planned - but it is inevitable that some will
+not be natively supported *yet*. Fortunately, while not as straightforward as
+:doc:`creating a new instrument<add_instrument>`, ResINS aims to make adding
+new :term:`versions<version>` as simple as possible:
+
+.. contents::
+    :backlinks: entry
+    :depth: 2
+    :local:
+
+
+How to add a version for personal use
+-------------------------------------
+
+Adding a :term:`version` to an :term:`instrument` only locally (without
+:ref:`submitting to ResINS<howto-version-resins>`) is the more difficult task
+since ResINS does not have an extension/add-ons mechanism. Therefore, to create
+a version, it is necessary to resort to either creating an entire instrument or
+working with Python dictionaries instead of YAML files:
```

Comment:
Do we want to implement an extension mechanism? This would be significantly more difficult since we would have make a way to accept multiple files and then combine them at runtime, so not sure if this is worth the effort